src/StackedChart.php

```diff
@@ -52,7 +53,7 @@ public function animations(array $animations): self
 
     public function title(string $title): self
     {
-        return $this->withMeta([ 'title' => $title ]);
+        return $this->withMeta([ 'title' => $title, 'uriKey' => Str::slug($title)]);
```

Comment:
![image](https://user-images.githubusercontent.com/9431885/82015882-791ec600-9688-11ea-9dd6-5fbe9f31dc47.png)
This change adds unique keys for the CardWrapper component. Before this I had "stacked-chart.undefined" for all Cards